hypothesis/via
# Redirect to Checkmate based error pages instead of serving our own

For: hypothesis/checkmate#15

This removes Via's own error pages and instead relies on whatever Checkmate tells us to show to people. We do a temporary redirect there for the moment, as we don't have any indication of how long the block should last for.

This should also not be merged until https://github.com/hypothesis/checkmate/pull/149 is done, which adds a version of the updated wording and feedback mechanism.

This cannot be merged until checkmatelib has been updated with: hypothesis/checkmatelib#5. The version number in the requirements is therefore a guess.

## Testing notes

 * Check out `checkmatelib` as a sibling to this project
 * In `checkmatelib` run: `git checkout add-html-view`
 * In `checkmatelib` run `make services dev`
 * Hack the `dev.txt` to replace the `checkmatelib` line with `-e ../checkmatelib`
 * Run `make dev`
 * Visit a blocked page: http://localhost:9080/http://bad.web.app
 * If you have the real rules loaded this should redirect (if not try `bad.example.com` instead)
